flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
Why is this still x86_64 only?

Reply:
I have not compiled and tested QRookie under other architectures, and I am not sure whether other dependencies will work under other architectures.